CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ version directory, and  then changes are introduced.
 ## [v6.1.1] 2020-05-07
 
 - Revert changes to deployment label selectors causing k8s-addons to fail.
+- Remove limits to calico-kube-controller
```

Comment:
This should go to `Unreleased` section.

Reply:
ouch. Thank you. I haven't seen that header.